src/utils.rs

```diff
@@ -480,6 +480,11 @@ pub fn integer_constant_from_i64(context: &MeliorContext, value: i64) -> Integer
     IntegerAttribute::new(uint256.into(), value)
 }
 
+pub fn integer_constant_from_i8(context: &MeliorContext, value: i64) -> IntegerAttribute {
```

Comment:
This should receive an `i8`, right?
```suggestion
pub fn integer_constant_from_i8(context: &MeliorContext, value: i8) -> IntegerAttribute {
```

Reply:
That's right